app/NEUFriends/models.py

```diff
@@ -0,0 +1,37 @@
+import uuid
+from django import forms
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
```

Comment:
Oh yeah, forgot to flag this: we want to be using Django's built-in User model, so we get authentication and views for free. Since we need our users to have extra fields though (bio, profile pic, etc.), [we need to extend it.](https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#extending-the-existing-user-model). Here's their docs about their auth system: https://docs.djangoproject.com/en/4.0/topics/auth/default/. Bottom-line is that we should delete this model and replace it with something like `AccountInfo` and follow the docs to extend the built-in user.